src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -1324,6 +1244,145 @@ public boolean isKnownParticipant( UUID organizationId, UUID studyId, String par
                 .collect( Collectors.toMap( pair -> pair.getLeft(), pair -> pair.getRight() ) );
     }
 
+    @Scheduled( fixedRate = STUDY_INFO_REFRESH_INTERVAL )
+    public void refreshStudyInformation() {
+        logger.info( "refreshing study information" );
+
+        try {
+            ApiClient apiClient = prodApiClientCache.get( ApiClient.class );
+            DataApi dataApi = apiClient.getDataApi();
+            SearchApi searchApi = apiClient.getSearchApi();
+            PrincipalApi principalApi = apiClient.getPrincipalApi();
+
+            // sync user
+            principalApi.syncCallingUser();
+
+            Map<UUID, Map<UUID, Map<String, UUID>>> allParticipants = Maps
+                    .newHashMap(); // orgId -> studyId -> participantId -> participantEKID
+            Map<UUID, Map<UUID, UUID>> studyEntityKeyIds = Maps.newHashMap(); // orgId -> studyId -> studyEKID
+            Map<UUID, Map<String, UUID>> deviceEntityKeyIds = Maps.newHashMap(); // orgId -> deviceId -> deviceEKID
+
+            Map<UUID, Map<CollectionTemplateTypeName, UUID>> orgEntitySets = entitySetIdsByOrgId
+                    .getOrDefault( CHRONICLE, Map.of() );
+
+            orgEntitySets.forEach( ( orgId, templateTypeESIDMap ) -> {
+
+                Map<UUID, UUID> studyIds = Maps
+                        .newHashMap(); // studyEKID -> studyId (for lookup when processing neighbors)
+
+                // entity set ids
+                UUID studiesESID = templateTypeESIDMap.getOrDefault( STUDIES, null );
+                UUID participantsESID = templateTypeESIDMap.getOrDefault( PARTICIPANTS, null );
+                UUID participatedInESID = templateTypeESIDMap.getOrDefault( PARTICIPATED_IN, null );
+                UUID usedByESID = entitySetIdsByOrgId.getOrDefault( CHRONICLE_DATA_COLLECTION, Map.of() )
```

Comment:
why are different entity sets being loaded in different ways? i.e. 
```
templateTypeESIDMap.getOrDefault( <collectionTypeName>, null );
```
vs
```
entitySetIdsByOrgId.getOrDefault( CHRONICLE_DATA_COLLECTION, Map.of() )
                        .getOrDefault( orgId, Map.of() ).getOrDefault( <collectionTypeName>, null );
```

Reply:
The participants, studies and participatedIn entity sets are in the `CHRONICLE` entity set collection.  perhaps It's cleaner to load all of them the same way.